### PR TITLE
Gameplay Page

### PR DESCRIPTION
**Pull request by marylou7**
I just added the basic content from Wikipedia. I plan to edit and shorten this as I work through the design and layout. I won't add a screenshot because there is nothing much to see yet.

### CODE REVIEW

**Comment by marylou7** [comment]
Does anyone know why my navbar is appearing like a vertical list? I copied Anas' code

pd screenshot

**Comment by ke444a** [comment]
Did you use the bootstrap template? In order for the bootstrap navigation bar to work, you need to type the necessary lines of code into your HTML document. As I see, you've not done that, so that's why the navigation bar does not look like it should be.

Bootstrap starter template can be found here: https://getbootstrap.com/docs/5.0/getting-started/introduction/#starter-template

**Comment by marylou7** [comment]
Ah ok. Thank you!

**Comment by marylou7** [comment]
Let me know if anyone wants to change anything to make it look better

https://user-images.githubusercontent.com/114414419/199349924-cfdad231-efd8-48e7-8efb-87f438ed7bca.mp4

File: gameplay_page.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+            </div>
+        </div>
+    </footer>
+</body>
+
+
+<!-- Option 1: Bootstrap Bundle with Popper -->
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"crossorigin="anonymous">
+</script>
```
Comment (ke444a) [inline gameplay_page.html:219]: Script tag should be added to the end of the body (INSIDE OF IT)

Moreover, please, add the space between attributes in script tag (there should be space before crossorigin attribute):
`crossorigin="anonymous" `

**Review by ke444a — COMMENTED** [review]
I think that the page looks good. However, there's one small error that I spotted in your HTML file that should be corrected. Besides that, well done!

File: gameplay_page.html (hunk 1 above)
Comment (marylou7) [reply gameplay_page.html]: Thanks, I just fixed those errors

**Pull request merged**